cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsRequestExecutor.java

```diff
@@ -203,7 +203,9 @@ private Object computeQueueLength(MetricsRequest metricsRequest) {
       if (queueName.isStream()) {
         streamNames.add(queueName.getSimpleName());
       } else if (queueName.isQueue()) {
-        String context = String.format("%s.f.%s.%s",
+        // TODO: After CDAP-1167 is fixed, namespace should be the first component of the queueName.
```

Comment:
Edit, CDAP-1167 got moved to 2.8. So we won't need a merge here. The update will come in in 2.8.